projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	for {
+		resp, err := http.Get("http://localhost:8080")
+		if err != nil {
+			fmt.Println("Server not working at the moment!")
+			fmt.Fprint(os.Stderr, "Server not working at the moment! ", err, "\n")
+			return
```

Comment:
What exit code will calling `return` from `main` use? What exit code do you probably want to use?